docs/api/version-history.md

```diff
@@ -20,6 +20,7 @@ keywords: "API, Docker, rcli, REST, documentation"
 * `POST /containers/create` now accepts additional values for the
   `HostConfig.IpcMode` property. New values are `private`, `shareable`,
   and `none`.
+* `POST /services/create` and `POST /services/(id)/update` now accept an `Isolation` field on container spec
```

Comment:
This should now be in v1.33 as v1.32 has shipped in a release; I opened https://github.com/moby/moby/pull/34908 to bump the API version